cmd/argo/commands/common/get.go

```diff
@@ -479,7 +479,7 @@ func printNode(w *tabwriter.Writer, node wfv1.NodeStatus, wfName, nodePrefix str
 	var args []interface{}
 	duration := humanize.RelativeDurationShort(node.StartedAt.Time, node.FinishedAt.Time)
 	if node.Type == wfv1.NodeTypePod {
-		podName := util.PodName(wfName, nodeName, templateName, node.ID, podNameVersion)
+		podName := util.PodName(wfName, node.Name, templateName, node.ID, podNameVersion)
```

Comment:
nodeName was formatted with the emojis etc to display that it was success full / not. 
Using util.PodName then computes the hash based off of this, resulting in the seemingly random numbers. 

This is also why the `argo get -o json/yaml ` output is totally fine

Reply:
I am personally of the opinion `PodName` is a bit misleading and should be renamed `GeneratePodName`, this implies the (potential) non-determinism better.

Reply:
feel free to make that change if you like :)